pipelines/working-with-pipelines-using-the-developer-perspective.adoc

```diff
@@ -7,19 +7,33 @@ include::modules/pipelines-document-attributes.adoc[]
 :context: working-with-pipelines-using-the-developer-perspective
 toc::[]
 
-You can use the *Developer* perspective of the {product-title} web console to create CI/CD Pipelines for your software delivery process while creating an application on {product-title}.
+You can use the *Developer* perspective of the {product-title} web console to create CI/CD Pipelines for your software delivery process.
```

Comment:
I will leave this as is. I have changed the next sentences. Changing this too would make it repetitive. Also, since it is an introduction and another option to create pipeline apart from the CLI way. You can seems alright.